Cargo.toml

```
@@ -26,6 +26,7 @@ rand_core = { version = "0.6.4", optional = true }
 rlp = { version = "0.5", optional = true, default-features = false }
 serdect = { version = "0.2", optional = true, default-features = false }
 zeroize = { version = "1", optional = true,  default-features = false }
+num-traits = "0.2"
```

As previously discussed in #158

```suggestion
num-traits = { version = "0.2", optional = true }
```